### PR TITLE
feat: named exports

### PR DESCRIPTION
**Pull request by PaulLeCam**
Following on https://github.com/ceramicnetwork/js-ceramic/pull/1661 this adds named exports for the DID resolvers and the core package.

### CODE REVIEW

File: packages/3id-did-resolver/src/index.ts [hunk 1]
```diff
@@ -216,7 +216,7 @@ const resolve = async (
   }
 }
 
-export default {
+export const threeId =  {
```
Comment (oed) [inline packages/3id-did-resolver/src/index.ts:219]:
```suggestion
export const threeIdDidResolver =  {
```

File: packages/key-did-resolver/src/index.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ const prefixToDriverMap: any = {
   0xed: ed25519,
 }
 
-export default {
+export const key = {
```
Comment (oed) [inline packages/key-did-resolver/src/index.ts:22]:
```suggestion
export const keyDidResolver = {
```

File: packages/3id-did-resolver/src/index.ts [hunk 3]
```diff
@@ -254,3 +254,5 @@ export default {
     },
   }),
 }
+
+export default threeId
```
Comment (oed) [inline packages/3id-did-resolver/src/index.ts:258]:
```suggestion
export default threeIdDidResolver
```

File: packages/key-did-resolver/src/index.ts [hunk 4]
```diff
@@ -55,3 +55,5 @@ export default {
     },
   }),
 }
+
+export default key
```
Comment (oed) [inline packages/key-did-resolver/src/index.ts:59]:
```suggestion
export default keyDidResolver
```

**Review by oed — APPROVED** [review]
Thanks. Made a naming suggestion.

**Comment by PaulLeCam** [comment]
@oed thanks for the review and suggestions, i changed it to exporting `getResolver()` directly, I think it's more consistent this way.

**Comment by oed** [comment]
Hm, so you would still have to use the default export if you are importing multiple resolvers then?

**Review by ukstv — APPROVED** [review]
And now it is proper `import * as KeyDidResolver from 'key-did-resolver'`. Nice.

**Comment by PaulLeCam** [comment]
> Hm, so you would still have to use the default export if you are importing multiple resolvers then?

No just need to alias the imports:

```js
import { getResolver as get3IDResolver } from '@ceramicnetwork/3id-did-resolver'
import { getResolver as getKeyResolver } from 'key-did-resolver'
```

**Pull request merged**